common/contenttypes.go

```diff
@@ -108,3 +108,23 @@ func (c ContentTypes) RemoveOverride(path string) {
 		}
 	}
 }
+
+func (c ContentTypes) CopyOverride(path, newPath string) {
```

Comment:
Exported symbol should have a godoc comment

Reply:
@gunnsth , got if fixed